pip installation fails: "nrt.h: No such file or directory"

cc @izaid 

From the end of the log (on Linux, also fails on OS X in a similar manner): 

```
    building 'numba.runtime._nrt_python' extension
    C compiler: gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC
    
    creating build/temp.linux-x86_64-2.7/numba/runtime
    compile options: '-Inumba -I/home/gmarkall/miniconda3/envs/piptest/lib/python2.7/site-packages/numpy/core/include -I/home/gmarkall/miniconda3/envs/piptest/include/python2.7 -c'
    gcc: numba/runtime/_nrt_python.c
    numba/runtime/_nrt_python.c:2:17: fatal error: nrt.h: No such file or directory
     #include "nrt.h"
                     ^
    compilation terminated.
    numba/runtime/_nrt_python.c:2:17: fatal error: nrt.h: No such file or directory
     #include "nrt.h"
                     ^
    compilation terminated.
    error: Command "gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Inumba -I/home/gmarkall/miniconda3/envs/piptest/lib/python2.7/site-packages/numpy/core/include -I/home/gmarkall/miniconda3/envs/piptest/include/python2.7 -c numba/runtime/_nrt_python.c -o build/temp.linux-x86_64-2.7/numba/runtime/_nrt_python.o" failed with exit status 1
    
    ----------------------------------------
Command "/home/gmarkall/miniconda3/envs/piptest/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-build-uFzAIr/numba/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-Ni6GUY-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-uFzAIr/numba
```

--- a/setup.py
+++ b/setup.py
@@ -156,6 +156,7 @@ setup(name='numba',
         "numba.typeconv": ["*.cpp", "*.hpp"],
         "numba.cuda.tests.cudadrv.data": ["*.ptx"],
         "numba.annotations": ["*.html"],
+        "numba.runtime": ["*.c", "*.h"]
       },
       scripts=["numba/pycc/pycc", "bin/numba"],
       author="Continuum Analytics, Inc.",

